cmd/goul/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:alpine AS build
+RUN apk add --update git gcc musl-dev libpcap-dev openssh-client
+
+WORKDIR /app
+COPY . .
+RUN go mod init nexclipper/goul
```

Comment:
I just committed go.mod and go.sum (to make it compatible with recent go) so this line can make an error. Could you please remove this line?